services/admin/src/server.ts

```diff
@@ -121,6 +123,20 @@ export function buildApp({ workspace }: { workspace: Workspace }): Application {
     }
   );
 
+  app.get<{ electionId: Id }, Admin.GetCvrsResponse>(
+    '/admin/elections/:electionId/cvrs',
+    (request, response) => {
+      response.json(
+        store
+          .getCastVoteRecordEntries(request.params.electionId)
+          .map(
+            (entry) =>
+              safeParseJson(entry.data).unsafeUnwrap() as CastVoteRecord
+          )
+      );
+    }
+  );
+
```

Comment:
My understanding is that this will return all the CVRs for an election. Which, on VxAdmin, could theoretically be several GBs worth of data. I'm not sure what our limitations are, but I'm wondering if we hit a scaling problem here eventually, especially considering write-in data. cc @eventualbuddha

Reply:
Hm, if we're talking multiple GBs of data, then it might be worth blocking this project on moving tally computation to the server first 😩

Do either of you have a sense of the upper limit of the data we're talking about here? Piecing together conversations I've overheard since joining, it sounds like that might be due to including image data in the CVR JSON? Or did I misunderstand that?

Reply:
cc @carolinemodic

As mentioned in eng planning, I'm thinking the options here are:

[S] Punt on reading raw CVR data from the backend and schedule some future time for moving tally computation to the backend.
[M] Invest in making sure this temporary API can handle large lists of CVRs (e.g. add support for paging)
[L?] Opportunistically move tally computation to the backend as part of this project and avoid investing in the temporary API.
Maybe (2) is a good compromise here, so we don't stay in the state of having two sources of truth for raw CVR data?

Reply:
Got a suggestion from @carolinemodic to strip out the image data from the `_ballotImages` field before returning the CVRs, in order to keep the response size low - this seems like a great stopgap before we eventually move tally computation server-side.

Did some more digging on http `content-length` limits and it does seem like some browsers have limits, but not chrome, AFAICT. All the same, I'm also going to add the express `compression` module, which reduces the ~730kB response in the PR description to 23kB (for 3000 CVRs)